alacritty/src/config/monitor.rs

```diff
@@ -69,26 +71,20 @@ pub fn watch(mut paths: Vec<PathBuf>, event_proxy: EventLoopProxy<Event>) {
         }
 
         loop {
-            let event = match rx.recv() {
-                Ok(event) => event,
+            let res = match rx.recv() {
+                Ok(res) => res,
```

Comment:
Let's just also log the error for `res` here and be done with it. Wasted too much time already on this dependency bump.

So both `Err(err)` and `Err(Err(err))` should print debug logs, just to make sure we're not ignoring serious issues.

Probably easiest to use one error message for both, no need to be that specific in a debug message unseen by most anyway.